mithril-common/src/digesters/dummy_immutable_db_builder.rs

```diff
@@ -76,11 +108,11 @@ impl DummyImmutablesDbBuilder {
         self
     }
 
-    /// Set the size of all files written by [build][Self::build] to the given `file_size` in bytes.
+    /// Set the size of all immutable files written by [build][Self::build] to the given `file_size` in bytes.
     ///
     /// Note: by default the size of the produced files is less than a 1kb.
-    pub fn set_file_size(&mut self, file_size: u64) -> &mut Self {
-        self.file_size = Some(file_size);
+    pub fn set_immutable_file_size(&mut self, file_size: u64) -> &mut Self {
```

Comment:
This is a bit ambiguous and does not clearly reflect the reality of an "immutable file":
- a chunk
- a primary index
- a secondary index

Reply:
Maybe something like `set_immutable_trio_file_size` would be better ?

Reply:
Done ✅